minio-operator.yaml

```diff
@@ -132,7 +147,7 @@ spec:
       serviceAccountName: minio-operator-sa
       containers:
         - name: minio-operator
-          image: minio/k8s-operator:1.0.8
+          image: nitishtiwari/operator:mirror
```

Comment:
is this left behind for testing during development?

Reply:
yes, fixed.